config/lib/express.js

```diff
@@ -55,9 +54,12 @@ module.exports.initLocalVariables = function (app) {
   app.locals.siteAnnouncement = config.siteAnnouncement || { enabled: false };
 
   // Assets
-  if (process.env.NODE_ENV !== 'production') {
+  if (process.env.NODE_ENV === 'production') {
+    app.locals.jsFiles = ['assets/main.js'];
+    app.locals.cssFiles = ['assets/main.css'];
```

Comment:
Can this be simplified further by just hardcoding these paths in rendered layout?

Reply:
It could, but I anticipate loading them dynamically in the future, currently we cannot enable webpack file splitting or hashed filenames, because we have no way of telling the backend which files to load.

The way to solve it is to have webpack output a json file listing all the asset files needed, and the backend can read this and render the layout correctly. So, that would keep it kind of similar to what is there, except loading the array from a json file.

Make sense?

Reply:
Yep! Sounds good. :-)